style.css

```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+body{
```

Comment:
Adding a space between `{` and the html element is good practice.

Reply:
Adding a space between { and the html element is good practice.

I suggest removing unused white spaces in the code. The code will be standardized and look cleaner